source/client-side-encryption/tests/README.rst

```diff
@@ -1817,7 +1819,7 @@ The command should be equivalent to:
 
 5. Using ``client_encryption``, create a data key with a ``local`` KMS provider and the keyAltName "def".
 
-Case 1: createKey()
+Case 1: createDataKey()
```

Comment:
Don't forget to keep the subsequent line consistent in length.

Reply:
_grumble grumble_ RST _grumble grumble_